### DISCUSSION TITLE
how to open snap.exe directly without presing enter?

### DISCUSSION THREAD

**Discussion by stvnkng** [wasiejen/Free-Snap-Tap#35] [Q&A] [answered]
my bat file look like this 
@echo off
.\free_snap_tap.exe ^
::-debug ^
::-tapfile=my_taps.txt ^
::-keyfile=my_keys.txt ^
-crossover=20 ^
-tapdelay=5,2 ^
-aliasdelay=5,2 ^
::-nomenu ^
::-nocontrols ^
::-nodelay ^
::-focusapp=Counterstrike ^
::
pause

**Comment by wasiejen** [comment, marked as answer]
Hi @stvnkng,
Simply remove the comment out "::" from the line -nomenu. Then it will start without the menu and directly started.

```bash
@echo off
.\free_snap_tap.exe ^
::-debug ^
::-tapfile=my_taps.txt ^
::-keyfile=my_keys.txt ^
-crossover=20 ^
-tapdelay=5,2 ^
-aliasdelay=5,2 ^
-nomenu ^
::-nocontrols ^
::-nodelay ^
::-focusapp=Counterstrike ^
::
pause
```

**Reply by stvnkng** [reply]
whats -focusapp=Counterstrike ^does? thx
